WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift

```diff
@@ -96,6 +96,8 @@ class PlanDetailViewController: UIViewController {
         planTitleLabel.text = plan.fullTitle
         planDescriptionLabel.text = plan.description
         planPriceLabel.text = priceDescriptionForPlan(plan)
+
+        purchaseButton.alpha = (plan == .Free) ? 0 : 1
```

Comment:
I'd suggest using `hidden` instead of `alpha`. Or even not having the button at all if it's a free purchase